Cython/Compiler/ExprNodes.py

```diff
@@ -13111,14 +13126,19 @@ def generate_operation_code(self, code, result_code,
                 result2 = operand2.py_result()
             else:
                 result2 = operand2.result()
+            special_bool_extra_args_result = [
+                extra_arg.result() for extra_arg in self.special_bool_extra_args
+            ]
+            special_bool_extra_args_result = ", ".join(special_bool_extra_args_result)
```

Comment:
Let's avoid using the same variable for two different things.
```suggestion
            special_bool_extra_args_result = ", ".join([
                extra_arg.result() for extra_arg in self.special_bool_extra_args
            ])
```